pkg/api/admin/openshiftcluster_convert.go

```diff
@@ -183,4 +183,9 @@ func (c *openShiftClusterConverter) ToInternal(_oc interface{}, out *api.OpenShi
 			Phase: api.InstallPhase(oc.Properties.Install.Phase),
 		}
 	}
+
```

Comment:
@jim-minter wdyt? Do we have a better solution? Pointers?

Reply:
Petr and I discussed this and agree it's the least bad of the options available.